pkg/utils/pod/scheduling.go

```diff
@@ -71,3 +71,15 @@ func ToleratesTaint(pod *v1.PodSpec, taint v1.Taint) bool {
 	}
 	return false
 }
+
+// ToleratesUnschedulable checks if the pod tolerates the special Kubernetes Unschedulable taint
+func ToleratesUnschedulable(spec *v1.PodSpec) bool {
```

Comment:
Can you re-use the ToleratesTaint function?

Reply:
I created this as it declares that we're checking if it tolerates the Kubernetes built-in Unschedulable taint. Additionally, it reads nicely to those who are checking out the code, rather than creating our own Taint as input that has the same values as the Unschedulable taint.

What do you think?